src/tests/Typing/TycoReg00014.exp

```diff
@@ -2,6 +2,7 @@
 // Return size: 0
 void fn00100000()
 fn00100000_entry:
+	ptr32 fp
```

Comment:
Should the pseudoregister `FramePointer` be excluded?

Reply:
I think it's OK. `fp` variable is used in followed code. So it should be declared.